python_bindings/tests/the_sort_function.c

```diff
@@ -1,92 +1,79 @@
 /*
- * Compile using 
+ * Compile using
  * gcc -std=c99 the_sort_function.c -shared -o the_sort_function.so
  */
 
-
+#include <stdbool.h>
 #include <stddef.h>
 #include <stdint.h>
-#include <stdbool.h>
 
+/**
+ *   The raw representation of an image passed around by generated
+ *   Halide code. It includes some stuff to track whether the image is
+ *   not actually in main memory, but instead on a device (like a
+ *   GPU). */
+struct buffer_t {
```

Comment:
two-space indentation. Also weird to declare buffer_t here, but I guess save that for a non-formatting PR.

Reply:
done